authors/apps/authentication/models.py

```diff
@@ -33,6 +36,16 @@ def create_user(self, username, email, password=None):
         user.set_password(password)
         user.save()
 
+        
+        
+        # send_mail(
```

Comment:
Remove the commented out code here.

Reply:
@inno-asiimwe Thanks, I've implemented this